app/models/score_manager.rb

```diff
@@ -59,6 +54,28 @@ def weekly_scores
     weekly_score_hash
   end
 
+  def calculate_day_score
+    @habits.each do |habit|
+      @day_score += habit.point_value
+    end
+  end
+
+  def calculate_date
+    if @num_days_ago > 0
+      date = @num_days_ago.days.ago.localtime
+    else
+      date = DateTime.now
```

Comment:
Useless assignment to variable - `date`.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.